client/lib/plans/constants.js

```diff
@@ -24,17 +25,23 @@ export const FEATURE_VIDEO_UPLOADS = 'video-upload';
 export const plansList = {
 	[ PLAN_FREE ]: {
 		getTitle: () => i18n.translate( 'Free' ),
-		getPriceTitle: () => i18n.translate( 'Free for life' )
+		getPriceTitle: () => i18n.translate( 'Free for life' ),
+		getProductId: () => 1,
+		getDescription: () => i18n.translate( 'Get a free blog and be on your way to publishing your first post in less than five minutes.' )
 	},
 
 	[ PLAN_PREMIUM ]: {
 		getTitle: () => i18n.translate( 'Premium' ),
-		getPriceTitle: () => i18n.translate( '$99 per year' )
+		getPriceTitle: () => i18n.translate( '$99 per year' ),
+		getProductId: () => 1003,
+		getDescription: () => i18n.translate( 'Your own domain name, powerful customization options, and lots of space for audio and video, and even Google AdWords credit.' )
```

Comment:
let's say `and $100 Google AdWords credit.` instead of `and even Google AdWords Credit.`